apps/events/tests/view_tests.py

```diff
@@ -559,6 +560,17 @@ def test_missing_access(self):
             'Du har ikke tilgang til å vise denne siden.', response)
         self.assertEqual(len(mail.outbox), 0)
 
+    def test_get_own_social_event_as_bedkom(self):
+        add_to_bedkom(self.user)
+        bedkom = Group.objects.get(name__iexact='bedkom')
+        event = generate_event(TYPE_CHOICES[0][0], organizer=bedkom)
+        url = reverse('event_mail_participants', args=(event.id,))
+
+        response = self.client.get(url)
+
+        self.assertEqual(response.context['event'], event)
+        self.assertEqual(len(mail.outbox), 0)
```

Comment:
Not sure why this test function should test that no emails are sent. I can see that some others do, but it should not be necessary.